tests/unit_tests/input_scan_tests.sh

```diff
@@ -1,6 +1,6 @@
 #! /bin/bash
 
-source ./entrypoint.sh "" "" "--debug"
+source ./entrypoint.sh "" "error" "--debug"
```

Comment:
I ran into situations like this, where I wanted to pass the argument directly to the entrypoint.sh file directly, but didn't pass arguments here because it affects all the test cases in this test suite. I understand that we need to have tests like this where we pass arguments to the entrypoint.sh file, but then we have to have separate files for them, and it gets messy. That's why I came up with the process_input_tests.sh file where we set the inputs directly.